miden-lib/src/tests/test_faucet.rs

```diff
@@ -688,3 +695,358 @@ fn test_get_total_issuance_succeeds() {
     )
     .unwrap();
 }
+
+#[test]
+fn test_faucet_contract_mint_fungible_asset_succeeds() {
+    let faucet_account_id = AccountId::try_from(ACCOUNT_ID_FUNGIBLE_FAUCET_ON_CHAIN).unwrap();
+    let faucet_account_code_src = include_str!("../../asm/faucets/basic.masm");
+    let faucet_account_code_ast = ModuleAst::parse(faucet_account_code_src).unwrap();
+    let mut account_assembler = Assembler::default()
+        .with_library(&MidenLib::default())
+        .expect("library is well formed")
+        .with_library(&StdLibrary::default())
+        .expect("library is well formed")
+        .with_kernel(SatKernel::kernel())
+        .expect("kernel is well formed");
```

Comment:
I think we have a function in `miden-lib` to build the assembler exactly with these parameters - so, we could probably use it instead of repeating this code.